### PR TITLE
First pass at contextual components.

### PR DESCRIPTION
**Pull request by toddjordan**

### CODE REVIEW

**Comment by toddjordan** [comment]
cc: @locks

File: source/localizable/components/wrapping-content-in-a-component.md [hunk 1]
```diff
@@ -59,3 +59,56 @@ form:
 It's important to note that the template scope inside the component
 block is the same as outside. If a property is available in the template
 outside the component, it is also available inside the component block.
+
+## Sharing Component Data with its Wrapped Content
+
+There is also a way to share data within your blog post component with the content its wrapping.
```
Comment (acorncom) [inline source/localizable/components/wrapping-content-in-a-component.md]: the content *it is* wrapping

File: source/localizable/components/wrapping-content-in-a-component.md [hunk 2]
```diff
@@ -59,3 +59,56 @@ form:
...
+In our blog post component,
```
Comment (acorncom) [inline source/localizable/components/wrapping-content-in-a-component.md]: No comma needed

**Comment by locks** [comment]
@homu r+

**Pull request merged**